tenprintcover.py

```diff
@@ -32,6 +32,9 @@
 
 import cairocffi as cairo
 
+PY2 = sys.version_info[0] == 2
+if PY2:
+    FileNotFoundError = IOError
```

Comment:
Good catch that [Py3’s FileNotFoundError](https://docs.python.org/3/library/exceptions.html#FileNotFoundError) (derived from `OSError`) is absent from Py2 which uses its own [IOError](https://docs.python.org/2/library/exceptions.html#exceptions.IOError) exception. I think this should be a separate pull request because it’s a separate issue.

However, redefining built-in Exception objects like above seems very hacky. Have you tried

```python
except (FileNotFoundError, IOError):
    ...
```